themes/grass/layouts/about/roadmap.html

```diff
@@ -0,0 +1,206 @@
+            <h3 class="mt-2 mb-2">Code quality and security</h3>
+            <p>We are committed to provide robust, secure and high quality
+              software by introducing tools to detect vulnerabilities, code
+              quality and formatting issues. To maintain high standards, we will:
+              <ul>
+                <li>Fix security and quality issues reported by the different
+                  tools.</li>
+                <li>Maintain and enhance CI workflows that help contributors
+                  comply with the standards.</li>
+              </ul>
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="section-sm bg-white mb-1">
+        <div class="row nm align-items-center">
+          <div class="col-2 text-center"><i class="fas fa-book fa-7x"></i></div>
+          <div class="col-10">
+            <h3 class="mt-2 mb-2">Documentation</h3>
+            <p>We are committed to offer complete and up to date documentation
+              for newcomers, users and contributors. Our goal is to:
+              <ul>
+                <li>Enhance tools' manuals by adding examples and figures where
+                  missing.</li>
+                <li>Convert html documentation to markdown to lower the barrier
+                  for contributions.</li>
+                <li>Provide tutorials on a wide variety of topics, both
+                  introductory and advanced.</li>
+              </ul>
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="section-sm bg-white mb-1">
+        <div class="row nm align-items-center">
+          <div class="col-2 text-center"><i class="fas fa-users
+          fa-7x"></i></div>
+          <div class="col-10">
+            <h3 class="mt-2 mb-2">Community growth</h3>
+            <p>Growing the GRASS GIS community is essential for fostering
+              innovation, collaboration, and the long-term sustainability of the
+              project. To grow and diversify our community, we aim to:
+              <ul>
+                <li>Foster community involvement by organizing yearly community
+                  meetings and promoting smaller local events.</li>
+                <li>Participate in mentoring programs (Google Summer of Code,
+                  GRASS student grants) to grow the base of users, contributors
+                  and promote inclusion.</li>
+              </ul>
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="section-sm bg-white mb-1">
+        <div class="row nm align-items-center">
+          <div class="col-2 text-center"><i class="fas fa-seedling
+          fa-7x"></i></div>
+          <div class="col-10">
+            <h3 class="mt-2 mb-2">Sustainability</h3>
+            <p>Different people have volunteered their time and money to sustain
+              and support GRASS GIS over time. To ensure GRASS' long-term success, we
+              aim to:
```

Comment:
I have the following suggestions: different->diverse, volunteered->invested, money->financial support, sustain and support...over time->ensure the ongoing development

```suggestion
            <p>A diverse group of contributors have invested their time and financial
             support to ensure the ongoing development of GRASS GIS. To ensure GRASS' long-term success, we
             aim to:
```